### ISSUE TITLE
how to calculate the Hessian of Image?

### ISSUE THREAD

**Issue by rancheng** [mariogeiger/hessian#2]
I want to calculate the hessian of a image, say, an ndarray like this:
```python
image = np.random.rand(50, 50)
image = torch.tensor(image)
hessian_img = hessian(image, [x, y])
```
should the `x` and `y` here be the `I_x` and `I_y`?

**Comment by Animadversio** [comment]
I guess you are talking about the hessian of `I`, i.e. pixel intensity as a function of position `I[x,y]` . 
Then you don't need this package at all. And autograd won't help you as image doesn't have a computational graph linked to x, y. 
Classic CV, i.e. 2d convolving the image with x, y gradient filter can do that.
